docs/current/partials/_install-sdk-go.md

```diff
@@ -5,7 +5,7 @@ The Dagger Go SDK requires [Go 1.15 or later](https://go.dev/doc/install).
 Install the Dagger Go SDK using the commands below:
 
 ```shell
-go get go.dagger.io/dagger@v0.3.0-alpha.1
+go get dagger.io/dagger@v0.3.0-alpha.1
```

Comment:
Need verification that `@v0.3.0-alpha.1` is still the correct version tag

Reply:
It probably won't be at launch time... Would it make sense to make this just `go get dagger.io/dagger` ?